src/test/java/plugins/OicAuthPluginTest.java

```diff
@@ -0,0 +1,309 @@
+package plugins;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsInAnyOrder;
+import static org.hamcrest.Matchers.in;
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertThrows;
+
+import dasniko.testcontainers.keycloak.KeycloakContainer;
+import jakarta.inject.Inject;
+import java.net.URL;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import org.jenkinsci.test.acceptance.junit.AbstractJUnitTest;
+import org.jenkinsci.test.acceptance.junit.WithPlugins;
+import org.jenkinsci.test.acceptance.po.GlobalSecurityConfig;
+import org.jenkinsci.test.acceptance.po.LoggedInAuthorizationStrategy;
+import org.jenkinsci.test.acceptance.po.OicAuthSecurityRealm;
+import org.jenkinsci.test.acceptance.po.WhoAmI;
+import org.jenkinsci.test.acceptance.utils.keycloack.KeycloakUtils;
+import org.json.JSONObject;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.keycloak.admin.client.Keycloak;
+import org.keycloak.admin.client.resource.GroupResource;
+import org.keycloak.admin.client.resource.RealmResource;
+import org.keycloak.representations.idm.ClientRepresentation;
+import org.keycloak.representations.idm.CredentialRepresentation;
+import org.keycloak.representations.idm.GroupRepresentation;
+import org.keycloak.representations.idm.RealmRepresentation;
+import org.keycloak.representations.idm.RoleRepresentation;
+import org.keycloak.representations.idm.UserRepresentation;
+import org.openqa.selenium.NoSuchElementException;
+
+@WithPlugins("oic-auth")
+public class OicAuthPluginTest extends AbstractJUnitTest {
+
+    private static final String REALM = "test-realm";
+    private static final String CLIENT = "jenkins";
+
+    private static final String KEYCLOAK_IMAGE="keycloak/keycloak:25.0.4@sha256:bf788a3b7fd737143f98d4cb514cb9599c896acee01a26b2117a10bd99e23e11";
```

Comment:
this is specifically not the latest to double check renovate confit.
was checked locally with `LOG_LEVEL=debug RENOVATE_CONFIG_FILE=/usr/src/app/.github/renovate.json renovate --platform=local` which did output the new versions, however I still want to see propose an update in CI.